use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -43,7 +43,13 @@ def module_names_parser(string: str) -> List[str]:
 
 
 if __name__ == "__main__":
-    arg_parser = argparse.ArgumentParser()
+    arg_parser = argparse.ArgumentParser(description="Showcase for the hybrid model converter.")
+    arg_parser.add_argument(
+        "--model-name",
+        default="gpt2",
+        type=str,
+        help="The name of the model to compile. Default is 'gpt2'.",
```

Comment:
Can you list the options the user has (which LLMs have we tried?) cc @fd0r

Reply:
We can't really have a list of all possibility here. Any model would work as long as the module to be converted is compatible.

Reply:
But it could just  be the models that you tried..

Reply:
We have a specific section in the readme about this with phi1.5 as an example. Adding more models here would also mean having there module names so it would make this docstring quite confusing. I would keep it with gpt2 and have the readme detail that you can use any kind of model.